scripts/test_line_bot.py

```
@@ -0,0 +1,233 @@
+        "こんにちは"
+    ]
+    for text in report_tests:
+        is_report = is_work_report(text)
+        print(f"  '{text}' -> {is_report}")
+    
+    print(f"\n✅ ユーティリティ関数テスト完了")
+
+
+def test_response_formatting():
+    """応答フォーマッティングのテスト"""
+    print(f"\n📝 応答フォーマッティングテスト")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)